packages/server/lib/modes/index.ts

```diff
@@ -5,10 +5,6 @@ import { makeDataContext } from '../makeDataContext'
 import random from '../util/random'
 
 export = (mode, options) => {
-  if (mode === 'record') {
```

Comment:
Why was this removed?

Reply:
It's actually dead code. There's a boolean in options that signifies record and we flex the run logic based on it.